integration-test/packages/atoms/simple-class/index.css

```diff
@@ -0,0 +1,10 @@
+.single-declaration {
```

Comment:
@sylvesteraswin the idea is to add an atom for each use case, e.g. global classes, nested selectors, pseudo-classes, vars, etc...

a single atom can have multiple classes that but they should be quite related to avoid having unrelated errors.

is a bit like the atom is the "describe" and the classes are the "it"